ros2topic/ros2topic/verb/list.py

```diff
@@ -34,19 +46,31 @@ def add_arguments(self, parser, cli_name):
         parser.add_argument(
             '--include-hidden-topics', action='store_true',
             help='Consider hidden topics as well')
+        parser.add_argument(
+            '-v', '--verbose', action='store_true',
+            help='list full details about each topic')
```

Comment:
Nit: The string should start with a capital letter, like the other help strings.

Reply:
OK